### PR TITLE
Learning continued

### PR DESCRIPTION
**Pull request by Samueljbk**
Uhhh I think this is how I do this I forget

### CODE REVIEW

File: src/scripts/github_activity_detector.py [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+load_dotenv()
+
+yeelight_ip = os.environ["BULB_IP"]
+bulb = Bulb(yeelight_ip)
+token = os.environ["PERSONAL_ACCESS_TOKEN"]
+
+g = Github(token)
+
+user = g.get_user("Samueljbk")
```
Comment (jerome3o) [inline src/scripts/github_activity_detector.py]: Add user as a env var

File: src/scripts/github_activity_detector.py [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+
+
+check_commit_status()
+
+while True:
+    current_time = datetime.datetime.now()
+    if current_time.hour == 0 and current_time.minute == 0:
+        set_light_colour("red")
+        time.sleep(60)
+    else:
```
Comment (jerome3o) [inline src/scripts/github_activity_detector.py]: I don't think this is needed as the resulting behaviour would probably be the same anyway

**Review by jerome3o — CHANGES_REQUESTED** [review]

File: src/scripts/github_activity_detector.py (hunk 1 above)
Comment (Samueljbk) [reply src/scripts/github_activity_detector.py]: Done

File: src/scripts/github_activity_detector.py (hunk 2 above)
Comment (Samueljbk) [reply src/scripts/github_activity_detector.py]: Done

**Review by jerome3o — APPROVED** [review]

**Pull request merged**